handlers/_swagger.js

```diff
@@ -171,6 +186,51 @@ function containsMultipartFormData(api) {
 
     return foundMultipartData;
 }
+/**
+ * The current use of this function is to figure out whether body parser needs to be configured
+ * This is the case when there is an operation that consumes form data without a file
+ * If there is a file, then multer will be used to parse form data
+ * @param api
+ * @returns {boolean}
+ */
+function containsUrlEncodedFormData(api) {
+    if (_.indexOf(api.consumes, 'application/x-www-form-urlencoded') > -1) {
```

Comment:
ignore this concern if all that is required is to know whether any path can consume this MIME type